tests/spec/w3c/headers-spec.js

```diff
@@ -231,7 +231,7 @@ describe("W3C — Headers", function() {
       const doc = await makeRSDoc(ops);
       expect($("#subtitle", doc).length).toEqual(0);
     });
-
+    
```

Comment:
This needs a cleanup (remove console.log, commented out code, etc.), but otherwise all good..

Reply:
cleaned up :+1: